modules/arch-service-catalog.adoc

```diff
@@ -0,0 +1,238 @@
+// Module included in the following assemblies:
```

Comment:
@bergerhoffer - can you review this section for accuracy and placement?

Reply:
Sure thing. @vikram-redhat @kalexand-rh I'll take a look at the content and will ask for some SME review of it as well to make sure that this all still applies for 4.0.

As for the placement, I might suggest to move it a little closer to Applications. Though I don't feel horribly strongly about it.

Reply:
@bergerhoffer, thanks! My only strong opinion about this PR is that it'll need rework after we get more feedback. After we have more content, I think it will be easier to structure the topics.

Reply:
@kalexand-rh I got confirmation that all of this is still applicable for 4.0 :).